usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -0,0 +1,227 @@
+from django import db
+from django.core.management.base import BaseCommand
+from pyspark.sql import SparkSession
+
+from usaspending_api.common.helpers.spark_helpers import (
+    configure_spark_session,
+    get_active_spark_session,
+    get_jdbc_connection_properties,
+    get_jdbc_url,
+    get_jvm_logger,
+)
+from usaspending_api.etl.management.commands.create_delta_table import TABLE_SPEC
+from usaspending_api.database_scripts.matview_generator.chunked_matview_sql_generator import (
+    make_copy_indexes,
+    make_copy_constraints,
+)
+
+SPECIAL_TYPES_MAPPING = {
+    db.models.UUIDField: {"postgres": "UUID USING {column_name}::UUID", "delta": "TEXT"},
+    "UUID": {"postgres": "UUID USING {column_name}::UUID", "delta": "TEXT"},
+    db.models.JSONField: {"postgres": "JSONB using {column_name}::JSON", "delta": "TEXT"},
+    "JSONB": {"postgres": "JSONB using {column_name}::JSON", "delta": "TEXT"},
+    db.models.DateTimeField: {"postgres": "TIMESTAMP WITH TIME ZONE", "delta": "TIMESTAMP WITHOUT TIME ZONE"},
+    "TIMESTAMP": {"postgres": "TIMESTAMP WITH TIME ZONE", "delta": "TIMESTAMP WITHOUT TIME ZONE"},
+}
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command reads data from a Delta table and copies it into a corresponding Postgres database (under a temp name).
+    As of now, it only supports a full reload of a table. All existing data will be deleted before new data is written.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--delta-table",
+            type=str,
+            required=True,
+            help="The source Delta Table to read the data",
+            choices=list(TABLE_SPEC.keys()),
```

Comment:
Not worth blocking over, but for a dictionary `list(<DICTIONARY>)` is the exact same thing as `list(<DICTIONARY_KEYS>)`.

Reply:
Good call. I think I copied that from another command but I don't see it in them anymore. Fixing.